.eslintrc.js

```diff
@@ -0,0 +1,74 @@
+module.exports = {
```

Comment:
@micheal-parks we should embark on a way to single source these from a repo akin to https://github.com/pion/.goassets

Reply:
Yeah we should probably publish our eslint / typescript rules [as an NPM package](https://eslint.org/docs/latest/developer-guide/shareable-configs) and add them as deps to the rest of our projects. Just made a ticket for myself.